CMSgov/onemac
# OY2-18208: SPA RAI on Dashboard to reference Medicaid vs CHIP

Story: https://qmacbis.atlassian.net/browse/OY2-18208
Endpoint: https://d3cyy6lbfnmuih.cloudfront.net/

### Details
This application started with Medicaid SPAs... hence, we used "spa" for a lot of things and they should be medicaidspa to differentiate with chipspa (users are getting confused)

### Changes
- Changed the component Type of the Medicaid SPAs from "spa" to "medicaidspa"
- Changed the component Type of the Medicaid SPA RAI Responses from "sparai" to "medicaidsparai"
- Updated the Type column in the UI to say "Medicaid SPA RAI"
- wrote migration script to convert existing data into new format.  This will run on develop and val, but not prod.  Expectation is that PROD will be migrated using the change-request table just before launch

### Implementation Notes
Changed constants, etc

### Test Plan
1. Login as a submitting user
2. Navigate to Submission Dashboard
3. SPA RAI type should now read Medicaid SPA RAI
